verticapy/connection/write.py

```diff
@@ -67,6 +90,34 @@ def delete_connection(name: str) -> bool:
     bool
         True if the connection was deleted,
         False otherwise.
+
+    Example
+    -------
+    .. ipython:: python
+        :suppress:
```

Comment:
@tconsigny 
I think we should change this because it may not work for every environment e.g. in my verticapylab, it throws an error obviously. Instead of executing, we can use a code-block here. Then you can just write the output manually.